arch/arm64/core/isr_wrapper.S

```diff
@@ -46,8 +46,22 @@ SECTION_FUNC(TEXT, _isr_wrapper)
 	bl	z_soc_irq_get_active
 #endif /* !CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER */
 
+#ifdef CONFIG_GIC_V3
```

Comment:
Is it better we use another macro, something like CONFIG_GIC_IGNORE_SPECIAL? Then CONFIG_GIC_V3 select this.
Because there could be GIC_V4, 5, 6... in the future, and possibly we still need this logic. :)

Reply:
would :
```
#if CONFIG_GIC_VER >= 3
```
be ok ?

Reply:
I think it's Ok